README.NewResourceFromTerraform.md

```diff
@@ -197,6 +197,34 @@ ServiceMappings file. Add the `ResourceConfig` for your resource:
         - remove_default_node_pool
     ```
 
+1. Add `mutableButUnreadableFields` if necessary. Fields may not be existent in the underlying GCP resource,
```

Comment:
@diviner524 Would you mind taking a look at "mutableButUnreadableFields" and "ignoredFields" sections, and let me know if that's accurate? I do not find detailed explanations on these two fields anywhere else, and I feel it's better to add it here.